### DISCUSSION TITLE
Hide the Device Presence Graph at the Top

### DISCUSSION THREAD

**Discussion by anjota80** [netalertx/NetAlertX#610] [Q&A]
On the original version of Pi.Alert, there was an option to remove the device presence graph at the top of the screen. Is there a way to remove the graph in this version? I have checked the settings and you can select what to show in the graph but cannot see an option to remove from the screen. If this is not an option, perhaps add this in for future development?
Thanks.

**Comment by jokob-sk** [comment]
Hi @anjota80 ,

The original version of PiAlert didn't have a presence chart, that was it wasn't shown in the first place. Anyway, the next version will have a setting to hide UI sections in the Devices pages:

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/78a67d7b-9e4e-47b7-aee8-e07708ad5829)


![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/42e1d0c2-7f13-4b17-97f2-651fc9df9811)

**Reply by anjota80** [reply]
Perfect. Thank you
